src/python/pants/testutil/BUILD

```diff
@@ -24,7 +24,6 @@ python_sources(
         "pants_integration_test.py": {
             "dependencies": ["//BUILD_ROOT:files", "src/python/pants/__main__.py"]
         },
-        "rule_runner.py": {"dependencies": ["//BUILD_ROOT:files"]},
```

Comment:
This is the fix that makes the errors in external repos visible in the pants repo.